HWC-Automation/pages/pharmacist.ts

```
@@ -0,0 +1,135 @@
+
+  private normalize(value?: string): string {
+    return (value || '').replace(/\s+/g, ' ').trim();
+  }
+
+  private escapeRegex(value: string): string {
+    return value.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+  }
+
+  private async getBeneficiaryNameFromFile(): Promise<string | undefined> {
+    try {
+      const fs: any = await (0, eval)("import('node:fs/promises')");
```

Biome (2.5.1)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)